ClinGen/clincoded
# Connects to #1784. Connects to #1786. New users, new affiliations, and bug fix in R22rc1.

**Technical notes:**
For #1786 testing, production data is required.

**Steps to test #1784:**
1. Log in and assign yourself to the **10030** affiliation.
2. Log out and log in again. In the affiliation modal dropdown menu, expect to see the **General Gene Curation EP** option.
3. Select an affiliation to proceed.
4. Go to `^/users` and enter `Babb` or `Mlynarski` into the **Filter table by** text field.
5. Expect to see an user record being returned for either `Larry Babb` and `Wojciech Mlynarski`.

**Steps to test #1786:**
1. Open the `DevTools` Console in your browser.
2. Go to `^/curation-central/?gdm=1d6fa43a-afc2-4b66-b892-51a944cd28a4`.
3. On this Gene-Disease Record page, expect to see the page being loaded as expected.
4. In the Console, there should be no error message pertinent to the following:
`TypeError: Cannot read property 'provisionalDate' of undefined at renderNewProvisionalStatus (render_new_provisional_status.js:27)`
